doc/doxygen/doxyfile.in

```diff
@@ -70,7 +74,9 @@ HIDE_IN_BODY_DOCS      = NO
 INTERNAL_DOCS          = NO
 CASE_SENSE_NAMES       = NO
 HIDE_SCOPE_NAMES       = NO
+HIDE_COMPOUND_REFERENCE= NO
 SHOW_INCLUDE_FILES     = YES
+SHOW_GROUPED_MEMB_INC  = NO
```

Comment:
I wonder if this should be YES instead

Reply:
Yes it should

Reply:
It was hard to see a difference for this config option. The only so-called member groups I can find (they have the `@{` doxygen tag) are either a misuse of the tag* or in `apps` (in-hand scanner), which isn't documented, and `metslib`, which is 3rdparty, so not documented. The ones properly defined and documented as doxygen modules have a `defgroup` somewhere, so `gpu`'s NPP_staging stuff (yay!), and again metslib, not documented. So the only affected thing is `NPP_*`, and surely enough it adds the include directions...but it's only to .hpp files... -.-"

Guess I'll go ahead and enable it, it's still better than nothing at all.

\*they use it right before the `namespace` keyword, which itself will group the stuff inside.